force-app/main/default/classes/NextStepFuture.cls

```diff
@@ -19,31 +19,43 @@
  */
 public with sharing class NextStepFuture {
 
-    /* 
-     * Question 1
-     * Update/Uncomment the template code to create a method
-     * Create a future method that updates all account and opportunities next step to 'Meeting in the future'
-     * Account Next Step field is a custom field
-     * Opportunity Next Step field is a standard field
-     * Update the corresponding test class AsynchronousTest and test method nextStepFuture_testUpdateAllNextStep to cover test this method
+    /*
+     * Future method to update all account and opportunities' next step to 'Meeting in the future'
      */
-    //[public | private ] [static] data_type method_name (input parameters) {
-    //  The body of the method
-    //}
+    @future
+    public static void updateAllNextStep() {
```

Comment:
This could would be an issue with larger data volumes. If you are trying to update every record in the database. You will quickly get to a situation where you have enough records in your instance to cause a limit exception. Future methods have expanded limits, but they still have limits. I know the instruction say to do this (I asked Warren to make an update here), but can you limit this to only the records you newly created (update your call to the method in the test as well)